cloud-foundry/nginx/conf/includes/redirect.conf

```diff
@@ -1,3 +1,48 @@
-rewrite ^/starters/task/maven/elston-ga$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.RELEASE/spring-cloud-task-app-descriptor-Elston.RELEASE.task-apps-maven redirect;
-rewrite ^/starters/task/docker/elston-ga$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.RELEASE/spring-cloud-task-app-descriptor-Elston.RELEASE.task-apps-docker redirect;
-rewrite ^/permanent-redirect$ https://spring.io permanent;
+rewrite ^/rabbitmq-maven-latest$ https://repo.spring.io/libs-release/org/springframework/cloud/stream/app/spring-cloud-stream-app-descriptor/Einstein.SR2/spring-cloud-stream-app-descriptor-Einstein.SR2.stream-apps-rabbit-maven redirect;
```

Comment:
It might be good to group stream and task app starter links as categories. All stream links one below the other and then tasks. It'll be easy to relate to.